unison-cli/tests/Unison/Test/CommandLine.hs

```diff
@@ -15,26 +15,10 @@ testCompletion compl CT {..} =
 test :: Test ()
 test = scope "commandline" $ do
   scope "completion" $ do
-    scope "completeWithinQueryNamespace" $ do
-      scope "only completes up to a single namespace boundary" $ do
-        testCompletion completeWithinQueryNamespace $
-          CT
-            { query = ".ba",
-              expected = [(".base", False)],
-              options = [".base", ".base.List", ".base.Map"]
-            }
-      scope "completes into the next namespace if query is a complete namespace" $ do
-        testCompletion completeWithinQueryNamespace $
-          CT
-            { query = ".base",
-              expected = [(".base", True), (".base.List", False), (".base.Map", False)],
-              options = [".base", ".base.List", ".base.Map"]
-            }
```

Comment:
I think we still want to specify the behavior for this query

Reply:
This combinator no longer exists and it's tougher to test the new one since it pulls options from sqlite, and does the filtering "by namespace" at the time it builds up the list of options. I still test the new `prefix` completion filter.